tests/broker/enchant_broker_set_ordering_tests.cpp

```diff
@@ -315,7 +315,7 @@ TEST_FIXTURE(EnchantBrokerSetOrdering_TestFixture,
 TEST_FIXTURE(EnchantBrokerSetOrdering_TestFixture,
 			 EnchantBrokerSetOrdering_EmptyLanguageTag_DoNothing)
 {
-	enchant_broker_set_ordering(_broker, "", "aspell,hunspell");
+	enchant_broker_set_ordering(_broker, "", "nuspell,aspell,hunspell");

```

Comment:
Please remove this change.

Reply:
Tha k you

Reply:
https://github.com/nuspell/enchant/commit/1739563777b88c5cb0203bb33ca77d784ca06d8b#diff-dad49e29d37a4ffc8e2e2a3ab77913aeR318